app/scripts/models/account.js

```diff
@@ -395,6 +395,8 @@ define(function (require, exports, module) {
         var email = this.get('email');
         var sessionToken = this.get('sessionToken');
 
+        this._metrics.logEvent('core.signin.attempt');
```

Comment:
Yeah, FWIU, the flow event should be propagated to DD already, I don't think this is needed either.